### DISCUSSION TITLE
Error 429

### DISCUSSION THREAD

**Discussion by FireWin93** [cloudbet/docs#64] [General]
Hi,
I get 429 errors on every bet I am making on the status api request. Is it an error that you know about?
Regards.

**Comment by kgravenreuth** [comment]
Hi @FireWin93 

This is a 429 rate limit response. We have defined rate limits as mentioned here: https://www.cloudbet.com/api/?urls.primaryName=Trading

```
Rate limit
Our place bet / bet status / bet history endpoint has rate limit - 1 request per second (1 RPS), we provide headers in response X-RateLimit-Limit and X-RateLimit-Remaining. Exceeding this rate limit will return HTTP 429 Too Many Requests response status code error.
```

So if you keep your requests within the defined rate limit values, then this issue should not be encountered.

If you still see such issues after reviewing your client implementation, then please let us know along with your Cloudbet username for further investigation. 

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
